tests/test_modules/test_builtin/test_childpart.py

```diff
@@ -154,3 +155,37 @@ def test_load_save(self):
         assert structure2 == expected
         self.p1.on_load(context, dict(sinkportConnector="blah_again"))
         assert b1.sinkportConnector.value == "blah_again"
+
+    def test_on_layout_first_call_sets_part_visibility_to_False(self):
```

Comment:
I'm not a big fan of mocked tests, I try to only use them to test the last few corner cases, most of the time I try and cover with integration tests like the rest of this file. I'll leave it up to you whether these add anything or not...

Reply:
Sure, I agree that it makes a lot of sense to use the integration test which now covers this check anyway.